public/robots.txt.production

```diff
@@ -0,0 +1,3 @@
+# Production
+User-agent: *
+Disallow:
```

Comment:
This actually means "disallow: nothing" in other words, "allow: everything".

http://www.robotstxt.org/robotstxt.html